### PR TITLE
Added Subscribe webpage

### PR DESCRIPTION
**Pull request by AaronStone24**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -2,33 +2,41 @@ require("dotenv").config();
...
 const connectDB = require("./config/dbConn");
 const errorHandler = require("./middleware/errorHandler.js");
 
 const PORT = process.env.PORT || 3500;
+const env = process.env.NODE_ENV || "dev";
 
 // connecting DB
-connectDB();
+if (env !== "dev")
+    connectDB();
```
Comment (pranjalbhatt1228) [inline server.js:17]: Should we not connect to DB for dev env? Should we make another DB model for the dev?

Comment (pranjalbhatt1228) [reply server.js:17]: Because we will need to test in dev env only

**Review by pranjalbhatt1228 — APPROVED** [review]

**Comment by pranjalbhatt1228** [comment]
LGTM

File: server.js (hunk 1 above)
Comment (AaronStone24) [reply server.js:17]: we could have more cases like, "dev-DB" for dev with DB

**Pull request merged**